src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -34,9 +41,35 @@ def ingest(
         defaults to None
     :param namespace: The namespace where the DAG will run
     """
+    source_list = [source] if isinstance(source, str) else list(source)
+    max_workers = None if num_batches else 20  # Default picked heuristically.
+
+    def build_io_uris() -> List[Tuple[str, str]]:
+        """Match input uri/s with output destinations
+
+        :param source: A sequence of paths or path to input
+        :param output_dir: A path to the output directory
+        """
+        vfs = tiledb.VFS(config=config)
```

Comment:
This has to be ` vfs = tiledb.VFS()` since it will use `/home/udf/.aws/config` to load credentials

Reply:
I have a tentative change to do this but have it separated out as its own rev pending investigating other issues.